agent/bench-scripts/postprocess/linpack-postprocess-cdm

```diff
@@ -30,7 +30,7 @@ if ($pp_mode eq 'cdm') {
     my $foo = 0;
 } else { # $pp_mode = html
     # Call the legacy post-process script, so we can build the static html reports later
-    system($ENV{'pbench_install_dir'} . "/bench-scripts/postprocess/linpack-postprocess `/bin/pwd` linpack default");
+    system($ENV{'pbench_install_dir'} . "/bench-scripts/postprocess/linpack-postprocess-sample `/bin/pwd` linpack default");
```

Comment:
You additionally need to modify the `agent/config/benchmark/linpack.json` (also it's mentioned in the `agent/Makefile` and in a bunch of unittests.

Reply:
Oh this is just `postprocess`, the comment belongs to the next commit instead...